packages/crypto/src/Ed25519e/Ed25519PublicKey.ts

```diff
@@ -30,8 +30,7 @@ export class Ed25519PublicKey {
    * @param message The original message the signature was computed from.
    * @returns true if the signature is valid; otherwise; false.
    */
-  async verify(signature: Ed25519Signature, message: HexBlob): Promise<boolean> {
-    await sodium.ready;
```

Comment:
Should we mention that `await Crypto.ready()` is a prerequisite before calling this function?

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1558/commits/0d664a6fb44ce29b61a0de3d456d676a8f43bd78